README.md

```diff
@@ -147,6 +147,39 @@ DELETE `/v1/s3/{account}/buckets/{bucket}
 | **409 Conflict**              | bucket is not empty             |
 | **500 Internal Server Error** | a server error occurred         |
 
+### Create a bucket user
+
+POST `/v1/s3/{account}/buckets/{bucket}/users
```

Comment:
Let's also add an example of what this is returning besides the code. That will be very helpful when trying to use the API.

Reply:
👍